zkvm/docs/zkvm-spec.md

```diff
@@ -1521,40 +1523,22 @@ _contract(P) bf prog_ **call** → _results..._
 Fails if either of the top two item is not a [data](#data-type) or
 the third-from-the-top is not a [contract](#contract-type).
 
+### select 
+_contract(P) X0 ... X{n-1}_ **select:_n_:_k_** → _contract(Xk)_
```

Comment:
since we use lowercase `n` and `k`, i'd keep the same lowercase notation instead of N throughout the spec below.